content/en/docs/installation/kubernetes.md

```diff
@@ -39,11 +39,11 @@ are included in a single YAML manifest file:
 Install the `CustomResourceDefinitions` and cert-manager itself:
 
 ```bash
-# Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.16.1/cert-manager.yaml
+# Kubernetes 1.16+
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.0/cert-manager.yaml
```

Comment:
The note below talks about `--validate=false` only being necessary if using kubectl < 1.15.4....so shall we remove that flag from our installation instructions now?

Reply:
In a followup PR, for the sake of not running another round of CI.